actions/hosts.last_checkin.yaml

```diff
@@ -0,0 +1,23 @@
+---
+name: hosts.last_checkin
+pack: foreman
+description: List all hosts that haven't reported back 
+runner_type: orquesta
+entry_point: workflows/hosts.last_checkin.yaml
+enabled: true
+parameters:
+  server:
```

Comment:
add in a `connection` parameter like in the rest of the actions, you can just pass this to `foreman.hosts.index`
```
 connection:
    type: string
    description: "Name of <connection> from this pack's configuration that specifies how to connect to a server."
required: false
```